ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -918,6 +918,47 @@ def test_paddle___lt__(
     )
 
 
+# __mod__
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="__mod__",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        num_arrays=2,
+        shared_dtype=True,
+        min_value=0,
+        exclude_min=True,
+    ),
+)
+def test_paddle___mod__(
```

Comment:
and also please remove the extra underscore.

Reply:
it's not an extra underscore, the method name was __mod__ which is why it has 3 underscores because the pattern is `test_paddle_<method>`